flexmeasures_openweathermap/__init__.py

```diff
@@ -48,6 +49,10 @@
         description=f"Name of the weather station asset, defaults to '{DEFAULT_WEATHER_STATION_NAME}'",
         level="debug",
     ),
+    "MINIMAL_DISTANCE_DEGREE": dict(
```

Comment:
Not sure why you switched maximal to minimal :-)
And I feel we should prefix with "OPENWEATHERMAP_", so it's clear in the config file where this belongs to.

Let's call it `OPENWEATHERMAP_MAXIMAL_DEGREE_LOCATION_DISTANCE`

(the setting `WEATHER_STATION_NAME` also should have this prefix, but let's not deal with that now)